.github/workflows/pythonapp.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will install Python dependencies and run tests with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python application
```

Comment:
Can we rename this to something that describes what this Github action does like 'Run unit tests'?